.github/workflows/llama-runtime-update.yml

```
@@ -0,0 +1,231 @@
+            io.llamacpp-manager.llama.cpp.build=${{ needs.detect.outputs.build_tag }}
+            io.llamacpp-manager.llama.cpp.image=${{ needs.detect.outputs.cuda_image }}
+          cache-from: type=gha,scope=llama.cpp-update-cuda
+          cache-to: type=gha,mode=max,scope=llama.cpp-update-cuda
+
+      - name: Publish summary
+        shell: bash
+        env:
+          RELEASE_TAG: ${{ needs.detect.outputs.release_tag }}
+          BUILD_TAG: ${{ needs.detect.outputs.build_tag }}
+          IMAGE_TAG: ${{ needs.detect.outputs.image_tag }}
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2016:info:10:8: Expressions don't expand in single quotes, use double quotes for that

(shellcheck)